.github/workflows/scripts/mesh_deploy.sh

```diff
@@ -0,0 +1,23 @@
+MESH=$1
+
+echo "Checking if mesheryctl is installed"
+	
+if mesheryctl
```

Comment:
Is this command can work well? If I remember right, we need to check the file to use `-f` parameter in shell.

```
if [ ! -f "path of mesheryctl" ]; then
        echo "Run"
    else
        echo "Not be found."
    fi
```

Reply:
🤔 I tried this method but mesheryctl is not found even though I have it installed. The path I supplied is `"/usr/local/bin/mesheryctl"`, I'm assuming that's the default location mesheryctl would be installed in ?

Reply:
Agree with you here, we can check the default path of `mesheryctl`. And I show you an example:

https://github.com/microsoft/vscode-dev-containers/blob/ced3b71afd3e9b74e3da1e32505144371e35ba28/containers/mit-scheme/.devcontainer/library-scripts/common-debian.sh#L54

The command can wrok well on check file.

Reply:
👍🏽  thanks for tip @Aisuko that worked